.github/workflows/test.yaml

```diff
@@ -19,9 +19,37 @@ jobs:
       - name: Setup go
         uses: actions/setup-go@v2
         with:
-          go-version: "1.21.0"
+          go-version: 1.21.0
```

Comment:
i'd keep the "" in place, if the version number goes to ".10" it will be a miss, and ".1" will be picked up